src/app/index.html

```diff
@@ -42,7 +42,7 @@
 	<div class="links">
 		<a href="https://github.com/AlexeyBoiko/DgrmJS" target="_blank">GitHub</a> | <a href="https://linkedin.com/in/alexey-boyko-tech" target="_blank">LinkedIn</a>
 	</div>
-	<svg id="diagram" xmlns="http://www.w3.org/2000/svg" style="touch-action: none; background-color: #fff; display:block;">
+	<svg id="diagram" xmlns="http://www.w3.org/2000/svg" style="touch-action: none; display:block;">
```

Comment:
Thanks for your help.
"background-color" here is used while svg -> png export. Without this ong background will be transparent.
I'll remove background-color from CSS